### ISSUE TITLE
Review HTML 5 sectioning elements and add ARIA landmark roles

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/docs-build#60]
Site has `header`, `nav`, `form`, and `footer` HTML 5 sectioning elements, but needs some additional sectioning elements and associated ARIA landmarks:

- add `main` section for each page
- make table of contents a `nav`
- make search on the homepage a `form`
- add ARIA landmark roles (`navigation`, `main`, `search`, and `content info`)
    - the search form on the docs pages already includes `<form role="search">`, so only the homepage search needs a role specified. 
- there are multiple `nav` sections in each page, so each `nav` should have an aria-label to distinguish its purpose (this includes the `nav` in the header, the table of contents, and the homepage search/filter section).

See the image below for an idea of how to implement ARIA landmark roles:
[W3C ARIA Landmark examples](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/index.html)

![aria-landmarks-docs-site](https://user-images.githubusercontent.com/15254041/54768160-fd84ce80-4bd4-11e9-8ba3-e6578d578a5e.jpg)

**Comment by HaSistrunk** [comment]
Issue is addressed in `homepage-redesign` branch and can be close when that branch is merged.
